### ISSUE TITLE
Firefox closes when I run swaymsg reload

### ISSUE THREAD

**Issue by xeruf** [manjaro-sway/manjaro-sway#401]
Maybe because it runs from X? But I think Emacs does that too...

**Comment by xeruf** [comment]
Interestingly only when the reload has no errors.

**Comment by boredland** [comment]
Should land [soon](https://bugzilla.mozilla.org/show_bug.cgi?id=1652820)

**Comment by boredland** [comment]
Strange things happen, but sway 1.8 does not seem to have this issue anymore:

[recording_20230102-205325.webm](https://user-images.githubusercontent.com/4662748/210273561-a77dbaf3-6ba6-4967-83e7-af51a528723a.webm)

If you'd like to upgrade:

```
yay -U https://archlinux.org/packages/community-testing/x86_64/sway/download/ https://archlinux.org/packages/community-testing/x86_64/wlroots/download/
```

**Comment by boredland** [comment]
will be fixed in 1.8
